pkg/logs/config/integration_config.go

```diff
@@ -112,9 +112,9 @@ func buildLogSourcesFromDirectory(ddconfdPath string) []*LogSource {
 			config := logSourceConfigIterator
 
 			// Users can specify tags as comma separated string, or as YAML array. Handle the first case here
-			if len(config.Tags) == 1 {
+			if len(config.Tags) > 1 {
 				newSlice := []string{}
-				for _, splitted := range strings.Split(config.Tags[0], ",") {
+				for _, splitted := range config.Tags {
```

Comment:
I am not sure this for loop is needed anymore, the trimming logic should be handled by the library I think.

Reply:
I gave this a try and it appears as though the lib does not trim the space